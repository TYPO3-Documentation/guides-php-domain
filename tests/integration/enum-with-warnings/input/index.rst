======================
PHP Enum with Warnings
======================

..  php:enum:: RedSuit : string
    :type: string

    In playing cards, a suit is one of the categories into which the
    cards of a deck are divided.

    ..  php:case:: Hearts
        :value: 'H'

        Hearts is one of the four suits in playing cards.

    ..  php:case:: Diamonds
        :value: 'D'

        Diamonds is one of the four suits in playing cards.


..  php:enum:: BlackSuit : string

    ..  php:case:: Clubs : 'C'

        Clubs is one of the four suits in playing cards.

    ..  php:case:: Spades : 'S'

        Spades is one of the four suits in playing cards.
