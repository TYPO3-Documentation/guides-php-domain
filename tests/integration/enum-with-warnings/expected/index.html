<!-- content start -->
    <div class="section" id="php-enum-with-warnings">
    <h1>PHP Enum with Warnings</h1>

            <dl class="php enum">
    <dt class="sig sig-object php"
        id="redsuit">
        <em class="property"><span class="pre">enum</span> </em>
        <span class="sig-name descname"><span class="pre">RedSuit</span></span>
<span class="pre"> : string</span> </dt>
    <dd>
        <p>In playing cards, a suit is one of the categories into which the
cards of a deck are divided.</p><dl class="php case">
    <dt class="sig sig-object php" id="redsuit-hearts">
        <em class="property"><span class="pre">case</span></em>
        <span class="sig-name descname"><span class="pre">Hearts</span></span>
 : <span class="pre">&#039;H&#039;</span> </dt>
    <dd><p>Hearts is one of the four suits in playing cards.</p></dd>
</dl>
<dl class="php case">
    <dt class="sig sig-object php" id="redsuit-diamonds">
        <em class="property"><span class="pre">case</span></em>
        <span class="sig-name descname"><span class="pre">Diamonds</span></span>
 : <span class="pre">&#039;D&#039;</span> </dt>
    <dd><p>Diamonds is one of the four suits in playing cards.</p></dd>
</dl>

    </dd>
</dl>

            <dl class="php enum">
    <dt class="sig sig-object php"
        id="blacksuit">
        <em class="property"><span class="pre">enum</span> </em>
        <span class="sig-name descname"><span class="pre">BlackSuit</span></span>
<span class="pre"> : string</span> </dt>
    <dd>
        <dl class="php case">
    <dt class="sig sig-object php" id="blacksuit-clubs-c">
        <em class="property"><span class="pre">case</span></em>
        <span class="sig-name descname"><span class="pre">Clubs : &#039;C&#039;</span></span>
 : </dt>
    <dd><p>Clubs is one of the four suits in playing cards.</p></dd>
</dl>
<dl class="php case">
    <dt class="sig sig-object php" id="blacksuit-spades-s">
        <em class="property"><span class="pre">case</span></em>
        <span class="sig-name descname"><span class="pre">Spades : &#039;S&#039;</span></span>
 : </dt>
    <dd><p>Spades is one of the four suits in playing cards.</p></dd>
</dl>

    </dd>
</dl>

    </div>

<!-- content end -->
